lib/index.js

```diff
@@ -19,6 +19,7 @@ var defaults = {
   debug: identity,
   lint: true,
   minify: false,
+  encapsulateStyles: true,
```

Comment:
I think it should be the default, this is a great feature to build isolated and predictable components. People should use custom properties (the API exposed by the component) and theme maps to tweak Components like you'd do with Web Components.
People can easily opt out if they have a good reason to do so.